Encode.xs

```diff
@@ -417,6 +443,8 @@ process_utf8(pTHX_ SV* dst, U8* s, U8* e, SV *check_sv,
     int check;
     U8 *d;
     STRLEN dlen;
+    char esc[80]; /* need to store UTF8SKIP * 6 + 1 */
```

Comment:
Thats a good idea, declaring `char esc[(sizeof("\\xHH") - 1) * UTF8_MAXBYTES + 1];` would make it very clear what the buffer is intended for.